settings.gradle.kts

```diff
@@ -37,6 +39,7 @@ include(":runtime:serde")
 include(":runtime:serde:serde-json")
 include(":runtime:serde:serde-xml")
 include(":runtime:serde:serde-form-url")
+include(":runtime:serde:serde-benchmarks")
 include(":runtime:protocol:http")
```

Comment:
**Question**: Why does this new test package belong in the **runtime** subproject? We have other non-publishing test packages like [**android-test**](https://github.com/awslabs/smithy-kotlin/tree/main/android-test) and [**compile-tests**](https://github.com/awslabs/smithy-kotlin/tree/main/compile-tests) as their own top-level projects. Seems like this could go at top-level or under a new **tests** subproject.

Reply:
Is there a reason not to? Having it close to what it tests seems like the most obvious location to me. The other two test projects aren't specific to one thing (also `android-test` is likely to just be deleted) which is why they don't live next to something more specific.

We could restructure it and have a root `tests` or `bechmarks` folder though. I think if we are to move it my preference would be for `benchmarks` folder with `serde-benchmarks` beneath that.

Reply:
One reason not to is to avoid inheriting config we don't want from parent projects, as you had to do for the new `skipPublish` configuration.

I think having a top-level **benchmarks** subproject makes sense (and even a top-level **tests** subproject). That'll facilitate a future where we can `./gradlew :benchmarks:all` (or something similar) to run every benchmark we have.

Reply:
That's the thing though we do want to inherit _almost_ all of that config. Also thats basically how gradle works anyway, you can just use an unqualified task name like `./gradlew jvmBenchmark` and it will run every task with that name.

But at any rate I'm fine moving it to a top level project under a `benchmarks` folder.